src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -1,3 +1,5 @@
+// noinspection HtmlUnknownTarget
```

Comment:
what's this?

Reply:
an accidental suppression comment from webstorm. I removed it